.github/workflows/CI.yaml

```diff
@@ -147,15 +164,43 @@ jobs:
 
       - name: "Test: Gear workspace"
         run: ./scripts/gear.sh test gear --release
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
 
       - name: "Test: JS metadata"
         run: ./scripts/gear.sh test js
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
 
       - name: "Test: Process node runtime testsuite"
         run: ./scripts/gear.sh test rtest
 
       - name: "Test: Runtime upgrade and queue processing"
```

Comment:
This is the only step that must be compiled in release mode because block production will take too long

Reply:
We need to build all artefacts in same mode to re-use artefacts and cache.

Reply:
I suggest to move this step into seperated job

Reply:
Disagree that this is the only one step. Runtime testsuit requires the same. And build in release everything for CI isn't the worst practice

Reply:
> Runtime testsuit requires the same

Why? 

> And build in release everything for CI isn't the worst practice

Isn't the worst but also isn't the best